package/network/services/dnsmasq/files/dnsmasq.init

```diff
@@ -539,8 +539,13 @@ dhcp_add() {
 	# Do not support non-static interfaces for now
 	[ static = "$proto" ] || return 0
 
+	ipaddr="${subnet%%/*}"
+	prefix="${subnet##*/}"
+
 	# Override interface netmask with dhcp config if applicable
-	config_get netmask "$cfg" netmask "${subnet##*/}"
+	config_get prefix2 "$cfg" netmask
```

Comment:
At the risk of being slightly too pedantic:
The variable that is read from UCI is a 'netmask' not a 'prefix'.
Imho, it should keep its name (`netmask`).
How about about calling the variable that it's merged into `netmask_or_prefix`?
Not pretty but more clear.

Other ideas:
`prefix` could be `runtime_prefix` and what is now `prefix2` could be `pool_config_netmask`.

Reply:
Okay, got a [definitive answer](https://github.com/openwrt/openwrt/pull/13879#issuecomment-1795624764) from @jow-.